recipe/meta.yaml

```diff
@@ -12,13 +12,12 @@ source:
 
 build:
   number: 0
-  # noarch: python
   skip: True      # [py<37]
 
 outputs:
   - name: {{ name }}
     build:
-      script: python -m pip install . -vv --no-deps
+      script: python -m pip install . -vv --no-deps --no-build-isolation
```

Comment:
Should this be moved to the `build` section that starts on [line 13](https://github.com/AnacondaRecipes/google-api-core-feedstock/pull/18/files#diff-f3725a55bf339595bf865fec73bda8ac99f283b0810c205442021f29c06eea9aR13)?